pkg/manifest/installer.go

```diff
@@ -46,16 +49,38 @@ const (
 	// cRDPollTimeout is the maximum wait time for all CRDs to be created.
 	cRDPollTimeout = 60 * time.Second
 
-	// operatorLabelStr indicates Istio operator is managing this resource.
-	operatorLabelStr = "istio-operator-managed"
 	// operatorReconcileStr indicates that the operator will reconcile the resource.
 	operatorReconcileStr = "Reconcile"
+)
+
+var (
+	// operatorLabelStr indicates Istio operator is managing this resource.
+	operatorLabelStr = name.ToOperatorNamespace("istio-operator-managed")
 	// istioComponentLabelStr indicates which Istio component a resource belongs to.
-	istioComponentLabelStr = "istio-component"
+	istioComponentLabelStr = name.ToOperatorNamespace("istio-component")
 	// istioVersionLabelStr indicates which Istio version
-	istioVersionLabelStr = "istio-version"
+	istioVersionLabelStr = name.ToOperatorNamespace("istio-version")
```

Comment:
It took me a minute to understand what was going on here. This might be more readable to define the well-known labels a const, e.g.

```golang
OperatorLabelOperatorManaged = "operator.istio.io/managed" // valid values are ...
OperatorLabelOperatorComponent = "operator.istio.io/component" 
OperatorLabelOperatorVersion = "operator.istio.io/version" 
```

@geeknoid created some comment packages to catalog annotations (see https://github.com/istio/pkg/blob/master/annotations/annotation.go). It might be worth doing the same for labels so that are more easily discover-able via docs.